build.gradle

```diff
@@ -130,14 +130,17 @@ subprojects {
                                 } else {
                                     depListNode = new Node(rootNode, 'dependencies')
                                 }
-                                project.configurations.shadow.allDependencies.each { dep ->
-                                    if (project.configurations.compile.allDependencies.find { compileDep -> compileDep.name == dep.name }) {
-                                        def depNode = new Node(depListNode, 'dependency')
-                                        depNode.appendNode('groupId', dep.group)
-                                        def prefix = dep.group == 'com.linecorp.decaton' && !dep.name.startsWith('decaton-') ? 'decaton-' : ''
-                                        depNode.appendNode('artifactId', prefix + dep.name)
-                                        depNode.appendNode('version', dep.version)
-                                        depNode.appendNode('scope', 'compile')
+
+                                project.configurations.compile.allDependencies.each { dep ->
```

Comment:
I think this needs to be `.runtime` rather than `.compile` to stay consistent with previous behavior?

Reply:
If we make it consistent with previous behavior, shouldn't be `compile` ?
https://github.com/line/decaton/pull/67/files#diff-c197962302397baf3a4cc36463dce5eaL134